HISTORY.rst

```diff
@@ -3,19 +3,25 @@
 History
 -------
 
+0.3.5 (2018-06-14)
+~~~~~~~~~~~~~~~~~~~~~
+
+* Fixed bug in which TypeError would result if a None value were mapped to a ``ListConfig``
```

Comment:
wrap `TyperError` around double backticks to render as code. same for `None`

Reply:
Done